docs/conversion.md

```diff
@@ -0,0 +1,64 @@
+| command           |           | YES         | [Pod.Spec.Container.Command](https://kubernetes.io/docs/api-reference/v1/definitions/#_v1_container)                                                                                                                                                                     |                                                                                                                             |
+| cgroup_parent     |           | UNSUPPORTED | N/A                                                                                                                                                                                                                                                                      | No compatibility with Kubernetes / OpenShift. Limited use-cases with Docker.                                                |
+| container_name    |           | YES         | Mapped to both [Metadata.Name](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/identifiers.md) and [Deployment.Spec.Containers.Name](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/identifiers.md) |                                                                                                                             |
+| deploy            | v3        | UNSUPPORTED |                                                                                                                                                                                                                                                                          | Upcoming support started                                                                                                    |
+| devices           |           | UNSUPPORTED | N/A                                                                                                                                                                                                                                                                      | Not supported within Kubernetes, see this [issue](https://github.com/kubernetes/kubernetes/issues/5607)                     |
+| depends_on        |           | UNSUPPORTED | N/A                                                                                                                                                                                                                                                                      | Elaborate on why this isn't supported and what happens when it's converted. Multiple users in #kompose has voiced concerns  |
```

Comment:
What concerns were voiced?
We have `depends_on` in unsupported keys. I'm not sure if that is right, maybe yes, maybe no :-D
It needs more explanation.
`depends_on` defines order in which containers should start. Problem is that there is no way to ensure that in Kubernetes. 
Questionis is if it matters or not :-)